webtool/fetch.py

```
@@ -339,7 +339,29 @@ def _mehrsprachig_aus_env():
     return None if roh is None else roh.strip().lower() in ("1", "true", "yes")
 
 
-def download_one(project: str, url: str) -> str:
+def _sprecher_aus_env(roh, i: int):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `_sprecher_aus_env`

(ANN202)